docs/src/main/twirl/play/soap/docs/template.scala.html

```diff
@@ -53,7 +53,7 @@ <h1 class="title"><a href="./Home.html">Play SOAP Documentation</a></h1><a href=
 							</div>
-							<p>© Lightbend 2019</p>
+							<p>© Lightbend Inc.</p>
```

Comment:
@renatocaval @dwijnand 
It's a footer of [documentation](https://playframework.github.io/play-soap/Home.html).
What needs to do with it?

Reply:
LGTM